scripts/md2html/build.sh

```
@@ -2,32 +2,86 @@
+  deploydir="deploy-preview"
+else
+  deploydir="deploy/oas"
+fi
+
+mkdir -p $deploydir/js
+mkdir -p $deploydir/temp
+cp -p node_modules/respec/builds/respec-w3c.* $deploydir/js/
+
+latest=$(git describe --abbrev=0 --tags)
+
+allVersions=$(ls -1 versions/[23456789].*.md | grep -v -e "\-editors" | sort -r)
```

Shellcheck (0.10.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)